### DISCUSSION TITLE
Documenting Interaction Design Decisions

### DISCUSSION THREAD

**Discussion by michaelchambersjnr** [ministryofjustice/moj-frontend#177] [EXPERIMENTAL: Components, pages and patterns]
For future development of the pattern library, could we brainstorm adding another layer where designers can document their design decisions. For example, what changed from one iteration to the next and how user pain points were address.

**Comment by kelliedesigner** [comment]
Latest discussion seems to point towards using GitHub Discussion threads to document the detail on design iterations in products, and use the Pattern Library to aggregate it into reusable guidance.

**Comment by kelliedesigner** [comment]
Template for documenting a new discussion to be trialed in communities:
https://docs.google.com/document/d/1LgF3YzKsyT-zPbQAumT5Zw3rp5fXDjghNSh67pw4dQo/edit?usp=sharing
